src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -105,4 +105,11 @@ describe('<Tabs />', () => {
       true
     );
   });
+
+  it('hides the heaer when there is no header', () => {
```

Comment:
typo: `heaer`

(But I still think that we shouldn't reuse the Panel component for the communication overlay. It will become even more obvious when you'll try to animate the enter/exit of the communication panel component)

Reply:
Updated the communication component to not to use Panel but I've left this test here.